embedder/embedder.go

```diff
@@ -226,6 +232,12 @@ func (p PlatformMessage) ExpectsResponse() bool {
 
 // SendPlatformMessage is used to send a PlatformMessage to the Flutter engine.
 func (flu *FlutterEngine) SendPlatformMessage(msg *PlatformMessage) Result {
+	flu.sync.Lock()
+	defer flu.sync.Unlock()
+	if flu.closed {
+		return ResultSuccess
```

Comment:
Just checking, is Success the result we should give back when shutting down?

Reply:
yeaa, we should create another result for this use-case.

Reply:
Wrapping [`m.engine.SendPlatformMessage(msg)`](https://github.com/go-flutter-desktop/go-flutter/blob/b21af7bbc3c034023d9cf4b86ea765e7097031dc/messenger.go#L41) into a [`tasker.do`](https://github.com/go-flutter-desktop/go-flutter/blob/b21af7bbc3c034023d9cf4b86ea765e7097031dc/messenger.go#L107-L112) could also solves to issue. Not a big fan of  `tasker.do` because it will make replies harder to implement.